runtime/calamari/src/lib.rs

```diff
@@ -531,8 +531,8 @@ parameter_types! {
 	pub const MaxDownwardMessageWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 10;
 }
 
-/// No local origins on this chain are allowed to dispatch XCM sends/executions.
-pub type LocalOriginToLocation = SignedToAccountId32<Origin, AccountId, RelayNetwork>;
```

Comment:
Old configuration meant that only the relay network's sovereign account could become this origin.
Now we've disabled even that, like Statemine/Statemint.